### DISCUSSION TITLE
Deprecated notification

### DISCUSSION THREAD

**Discussion by obiscr** [obiscr/ChatGPT#239] [Announcements]
# Important

The ChatGPT Tab and Online ChatGPT Tab will be removed in the next release. Please use the “EasyCode” tab instead.

**Comment by yalexx** [comment]
EasyCode dont have chat gpt 4

**Comment by obiscr** [comment]
> EasyCode dont have chat gpt 4

We are planning future new features of EasyCode Tab. Updates will be posted as they become available. Please stay tuned. Thank you very much!
